src/picongpu/include/particles/InterpolationForPusher.hpp

```diff
@@ -33,6 +29,13 @@ namespace picongpu
 using namespace PMacc;
```

Comment:
Looks like I have not seen this in the old pull request.
Could you please remove this. If this is needed for an object than please use it explicit.